packages/ui/src/components/va-stepper/VaStepperControls.vue

```diff
@@ -1,23 +1,12 @@
 <template>
   <div class="va-stepper__default-controls">
-    <va-button
-      preset="primary"
-      :disabled="$props.modelValue <= 0"
-      @click="$props.stepControls.prevStep()"
-    >
+    <va-button preset="primary" :disabled="$props.modelValue <= 0" @click="$props.stepControls.prevStep()">
```

Comment:
This file seems to be changed with prettier only, so we can revert this changes?